internal/operators/manager_test.go

```diff
@@ -834,7 +834,7 @@ var _ = Describe("Operators manager", func() {
 				// lso isn't part of any bundle
 				Expect(bundle.Operators).NotTo(ContainElement(lso.Operator.Name))
 				if bundle.Name == operatorscommon.BundleVirtualization {
-					Expect(bundle.Operators).To(ContainElements(odfOperator.GetName(), cnvOperator.GetName()))
+					Expect(bundle.Operators).To(ContainElements(cnvOperator.GetName()))
```

Comment:
Consider replacing odf with mtv to keep the test logic (getting a bundle with few operators....)

Reply:
Done, thanks @eranco74.